tybalt/models.py

```diff
@@ -105,15 +108,31 @@ def _connect_layers(self):
         _x_decoded_mean = self.decoder_model(decoder_input)
         self.decoder = Model(decoder_input, _x_decoded_mean)
 
-    def train_vae(self, train_df, test_df):
+    def train_vae(self, train_df, test_df, separate_loss=False):
```

Comment:
I am not particularly qualified to comment on python conventions, but I would want more information about what `separate_loss` is doing here